test/services/school/index.test.js

```diff
@@ -41,7 +42,51 @@ describe('school service', () => {
 			sampleSchoolData = await School.findById(school._id).lean().exec();
 			delete sampleSchoolData._id;
 		});
+		it('should be possible to see only limited amount of fields if the user is not logged in', async () => {
+			const params = {
+				provider: 'rest',
+				headers: {
+					authorization: undefined
+				},
+				account: undefined,
+			}
+			const result = await schoolService.find(params);
+			const expectedFields = ['purpose', 'name', 'id', 'years', 'isTeamCreationByStudentsEnabled'];
+			const notExpectedFields = ['fileStorageType', 'documentBaseDir', 'inMaintenance', 'currentYear', 'systems', 'federalState'];
```

Comment:
Codacy found an issue: [This line has a length of 136. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5734878)